specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,446 @@
+          "NotebookWorkspacesResource"
+        ],
+        "operationId": "NotebookWorkspaces_Start",
+        "x-ms-examples": {
+          "CosmosDBNotebookWorkspaceStart": {
+            "$ref": "./examples/CosmosDBNotebookWorkspaceStart.json"
+          }
+        },
+        "description": "Starts the notebook workspace",
+        "parameters": [
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted. The start operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NotebookWorkspaceCreateParameters": {
+      "description": "Parameters to create a notebook workspace resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+        }
+      ]
+    },
+    "NotebookWorkspaceListResult": {
+      "description": "A list of notebook workspace resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Array of notebook workspace resources",
+          "items": {
+            "$ref": "#/definitions/NotebookWorkspace"
+          }
+        }
+      }
+    },
+    "NotebookWorkspace": {
+      "description": "A notebook workspace resource",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NotebookWorkspaceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ARMProxyResource"
+        }
+      ]
+    },
+    "NotebookWorkspaceProperties": {
+      "description": "Properties of a notebook workspace resource.",
+      "type": "object",
+      "properties": {
+        "notebookServerEndpoint": {
+          "description": "Specifies the endpoint of Notebook server.",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
```

Comment:
Does it mean `provisioningState`? Just a sharing, there are some logic rely on the word `provisioningState` in the runtime of SDKs. If the `status` is the same thing with `provisioningState`, it's better to use the word `provisioningState`.

Reply:
This is different than provisioning state.